mslib/msui/msui_mainwindow.py

```diff
@@ -50,8 +50,7 @@
 from mslib.msui.icons import icons, python_powered
 from mslib.utils.qt import get_open_filenames, get_save_filename, show_popup
 from mslib.utils.config import read_config_file, config_loader
-from mslib.utils import release_info
-from PyQt5 import QtGui, QtCore, QtWidgets
+from PyQt5 import QtGui, QtCore, QtWidgets, QtTest
```

Comment:
ToDo: QtTest is only available for developers. It is not in the released package. The feature needs to become differently implemented.

Reply:
https://github.com/Open-MSS/MSS/pull/2551